### PR TITLE
[master ]Update dependency versions for update 10

### PR DESCRIPTION
**Pull request by lnash94**
## Purpose

Fixes:

## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -10,50 +10,50 @@ researchgateReleaseVersion=2.8.0
 testngVersion=7.6.1
 ballerinaGradlePluginVersion=2.0.1
 
-ballerinaLangVersion=2201.9.0
+ballerinaLangVersion=2201.10.0-20240806-083400-aabac46a
 
-stdlibSqlVersion=1.13.0
+stdlibSqlVersion=1.13.2
```
Comment (TharmiganK) [inline gradle.properties:15]: Need to update the new timestamped version right?

Comment (lnash94) [reply gradle.properties]: yeah, updated with the new timestamp version

**Comment by daneshk** [comment]
@lnash94 can you update the distribution version here https://github.com/ballerina-platform/module-ballerinax-mysql/blob/master/build-config/resources/Ballerina.toml#L10

**Comment by lnash94** [comment]
> @lnash94 can you update the distribution version here https://github.com/ballerina-platform/module-ballerinax-mysql/blob/master/build-config/resources/Ballerina.toml#L10

Please review I have updated

**Review by TharmiganK — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**